dagql/idtui/frontend.go

```diff
@@ -406,6 +417,7 @@ func (fe *Frontend) initZoom(v *progrock.Vertex) {
 	} else {
 		vt = midterm.NewTerminal(fe.window.Height, fe.window.Width)
 	}
+	vt.Raw = true
```

Comment:
Previously midterm would always interpret `\n` as `\r\n` which is needed for rendering simple logs, but was incorrect for displaying TUIs like `vim` or `htop`. Now you have to opt in to that behavior.